FreeRTOS-Plus/Source/Application-Protocols/network_transport/freertos_plus_tcp/using_plaintext/using_plaintext.c

```diff
@@ -182,5 +185,15 @@ int32_t Plaintext_FreeRTOS_send( NetworkContext_t * pNetworkContext,
         socketStatus = 0;
     }
 
+    #if ( configUSE_PREEMPTION == 0 )
+        {
+            /* Allow the IP task to instantly send the packets through the ethernet interface.
```

Comment:
Since this code is not related to MQTT, I'd suggest to update the comment to the following:

```
FreeRTOS_send adds the packet to be sent to the IP task's queue for later processing.
The packet is sent later by the IP task. When FreeRTOS is used in collaborative
mode (i.e. configUSE_PREEMPTION is 0), call taskYIELD to give IP task a chance to run
so that the packet is actually sent before this function returns.
```

Reply:
Done.